Library/Homebrew/cmd/install.rb

```diff
@@ -21,8 +21,12 @@
 #:
 #:    If `--cc=`<compiler> is passed, attempt to compile using <compiler>.
 #:    <compiler> should be the name of the compiler's executable, for instance
-#:    `gcc-4.2` for Apple's GCC 4.2, or `gcc-4.9` for a Homebrew-provided GCC
-#:    4.9.
+#:    `gcc-8` for gcc 8, `gcc-4.2` for Apple's GCC 4.2, or `gcc-4.9` for a
+#:    Homebrew-provided GCC 4.9. In order to use LLVM's clang, use
+#:    `llvm_clang`. To specify the Apple-provided clang, use `clang`. This
+#:    parameter will only accept compilers that are provided by Homebrew or
+#:    bundled with MacOS. Please do not file issues if you encounter errors
```

Comment:
macOS

Reply:
I've been spelling it wrong all this time...O.o